src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -102,15 +132,26 @@ const QuickMarcEditor = ({
     );
 
     const end = (
-      <Button
-        buttonStyle="primary mega"
-        disabled={saveFormDisabled}
-        id="quick-marc-record-save"
-        onClick={confirmSubmit}
-        marginBottom0
-      >
-        <FormattedMessage id="stripes-acq-components.FormFooter.save" />
-      </Button>
+      <>
+        <Button
+          buttonStyle="default mega"
+          disabled={saveFormDisabled}
+          id="quick-marc-record-save-edit"
+          onClick={(event) => confirmSubmit(event, true)}
+          marginBottom0
+        >
+          <FormattedMessage id="stripes-acq-components.FormFooter.save.edit" />
```

Comment:
Looks like this translation doesn't exist in `stripes-acq-components`.
Let's create translations for both buttons in `ui-quick-marc` so we don't depend on this other module

Reply:
Yes, I forgot it. I have a permission problem to commit into `stripes-acq-components` module.

Reply:
Yeah, it's not modules so we need to ask for permissions to push changes there. It'll be much simpler to add our own translations